vlib/x/json2/json_module_compatibility_test/json_todo_test.vv

```diff
@@ -220,8 +207,7 @@ fn test_nested_type() {
 			}
 		}
 	}
-	out := json.encode(data)
-	assert out == data_expected
+
```

Comment:
why delete these?

Reply:
It is a .vv file. I remove from there to be put at .v file